src/app/Instance/BridgeOverview/Components/OBEmptyState.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Button,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  Text,
+  TextContent,
+  Title,
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+import React from "react";
+interface OBEmptyStateProps {
+  title: string;
+  description: string;
+  buttonName: string;
+  changeState?: () => void | undefined;
+  variant:
+    | "link"
+    | "primary"
+    | "secondary"
+    | "tertiary"
+    | "danger"
+    | "warning"
+    | "plain"
+    | "control"
+    | undefined;
```

Comment:
I get these are values for PF `Button` variants. You could do instead

```suggestion
  variant: ButtonProps["variant"]
```

And then import `ButtonProps` from PF.